rtgui/previewhandler.cc

```diff
@@ -204,27 +204,19 @@ Glib::RefPtr<Gdk::Pixbuf> PreviewHandler::getRoughImage (int x, int y, int w, in
             h = image->getHeight() * totalZoom;
         }
 
-        int ix = x * zoom;
-        int iy = y * zoom;
+        x *= zoom;
+        y *= zoom;
 
-        if (ix < 0) {
```

Comment:
I see. (Asserts might still be useful to get a clean crash in debugging builds when this invariant is violated due to an error in other parts of the code, but it is not strictly necessary.)